pkg/proxy/iptables/proxier.go

```diff
@@ -1447,7 +1447,7 @@ func (proxier *Proxier) syncProxyRules() {
 				args = append(args, "-m", "recent", "--name", string(endpointChain), "--set")
 			}
 			// DNAT to final destination.
-			args = append(args, "-m", protocol, "-p", protocol, "-j", "DNAT", "--to-destination", readyEndpoints[i].Endpoint)
+			args = append(args, "-m", protocol, "-p", protocol, "-j", "DNAT", "--to-destination", endpoints[i].Endpoint)
```

Comment:
this line is wrong in the iptables commit and then fixed in the ipvs commit; the fix needs to be squashed back into the iptables commit